engine/flink/kafka-util/src/main/scala/pl/touk/nussknacker/engine/kafka/exception/KafkaExceptionSerializationSchema.scala

```diff
@@ -0,0 +1,69 @@
+package pl.touk.nussknacker.engine.kafka.exception
+
+import io.circe.generic.JsonCodec
+import io.circe.syntax.EncoderOps
+import org.apache.kafka.clients.producer.ProducerRecord
+import pl.touk.nussknacker.engine.api.MetaData
+import pl.touk.nussknacker.engine.api.exception.{EspExceptionInfo, NonTransientException}
+
+import java.io.{PrintWriter, StringWriter}
+import java.net.InetAddress
+import java.nio.charset.StandardCharsets
+import java.time.ZoneId
+import scala.io.Source
+
+case class KafkaExceptionSerializationSchema(metaData: MetaData, consumerConfig: KafkaExceptionConsumerConfig) {
```

Comment:
I wonder why assume errors will be always log as json? What about log errors as avro, etc?

Reply:
I added comment to add further formats in the future